src/components/Layout/Header.tsx

```diff
@@ -1,5 +1,40 @@
-export const Header: React.FC = () => (
-  <header className="fixed w-full bg-white">
-    <p className="px-4 py-8 text-3xl md:px-8">SportujSpolu</p>
-  </header>
-);
+'use client';
+import { useSession, signOut } from 'next-auth/react';
+import Link from 'next/link';
+
+export const Header: React.FC = () => {
+  const { data: session } = useSession();
+
+  return (
+    <header className="fixed w-full bg-white">
+      <nav className="flex flex-row items-center justify-between">
+        <Link href="/" className="px-4 py-8 text-3xl md:px-8">
+          SportujSpolu
+        </Link>
+        <p className="px-4 md:px-8">
+          <b>{session?.user?.name} </b>
+          <i>{session?.user?.email}</i>{' '}
+          {session?.user ? (
+            <button
+              className="pl-5 hover:text-green-500"
+              onClick={() => signOut()}
+            >
+              Sign out
+            </button>
+          ) : (
+            <>
+              {' '}
```

Comment:
Remove {' '}. Prettier inserted {' '} to preserve whitespace between elements, but it does not need to be there.